docs/en/latest/plugins/response-rewrite.md

```diff
@@ -118,6 +118,45 @@ X-Server-balancer_addr: 127.0.0.1:80
 {"code":"ok","message":"new json body"}
 ```
 
+Replace the string client_addr of body to client_addr_replace.
+
+```shell
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+  "plugins": {
+    "response-rewrite": {
+            "headers": {
+                "X-Server-id": 3,
+                "X-Server-status": "on",
+                "X-Server-balancer_addr": "$balancer_ip:$balancer_port"
+            },
+            "filters": [{"regex": "client_addr", "scope": "global", "replace": "client_addr_replace"}],
+            "vars":[
+                [ "status","==",200 ]
+            ]
+    }
+  },
+  "upstream": {
+    "type": "roundrobin",
+    "nodes": {
+      "127.0.0.1:8881": 1
```

Comment:
`httpbin.org` is recommended, no one knows what `127.0.0.1:8881` will respond to.

Reply:
updated

Reply:
LGTM